tests/Integration/Behaviour/Features/Scenario/CatalogPriceRule/catalog_price_rules_listing.feature

```diff
@@ -0,0 +1,305 @@
+      | catalog price rule 1  | usd        | UnitedStates  | visitor  | 1              | amount        | 111.50          | testShop | true        | 50    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 2  | usd        | UnitedStates  | visitor  | 1              | amount        | 50              | testShop | true        | 10    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+    And I should be able to see following list of catalog price rules with language "en" with limit 50 offset 0 and total 5 and product "product1":
+      | name                 | currency   | country       | group    | from quantity  | reduction type| reduction value | shop     | includes tax| price | from                | to                  |
+      | catalog price rule 1 | usd        | UnitedStates  | visitor  | 1              | amount        | 111.50          | testShop | true        | 50    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 2 | usd        | UnitedStates  | visitor  | 1              | amount        | 50              | testShop | true        | 10    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 3 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 4 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 5 | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+    And I should be able to see following list of catalog price rules with language "en" with limit 50 offset 2 and total 5 and product "product1":
+      | name                  | currency   | country       | group    | from quantity  | reduction type| reduction value | shop     | includes tax| price | from                | to                  |
+      | catalog price rule 3  | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 4  | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+      | catalog price rule 5  | usd        | UnitedStates  | visitor  | 5              | amount        | 10              | testShop | true        | 20    | 0000-00-00 00:00:00 | 0000-00-00 00:00:00 |
+
+  Scenario: I can see correctly with category condition
+    When I add following conditions to catalog price rule "catalogPriceRuleReference1":
+      | type     | value |
+      | category | women |
+    And I assign product product1 to following categories:
```

Comment:
I think it's safer to create another product specifically for this category condition test Because `product1` is also refused as the one having the correct conditions for suppliers later so it becomes mixed with multiple characteristics that could match the condition

If we have a dedicated product which is the only one with this category then it makes the test more stable, the same principle should be reused for other conditional tests (a product with matching suppliers, manufacturer, feature, ...) The maybe the "product2" you sed as a reference (with the rules decreasing as they get more and more conditions) will not be that useful anymore since product1 could be enough to keep as a reference

Reply:
Except for this detail that all these behat scenarios are very detailed, good job!